docs/_includes/ide-tabs.html

```diff
@@ -0,0 +1,26 @@
+<!-- Uses the method selectTabs defined in ide-tabs.js -->
+<div class="shadow" markdown="0">
+
+    <ul class="nav nav-tabs">
+        <li class="nav-item">
+            <a class="nav-link eclipse active" onclick="selectTabs( 'eclipse' );" id="eclipse-tab" data-toggle="tab" href="#">
+                <img alt="Eclipse logo" src="{{ '/assets/images/logo/eclipse-logo.png' | relative_url }}" /> Eclipse
+            </a>
+        </li>
+        <li class="nav-item">
+            <a class="nav-link intellij" onclick="selectTabs( 'intellij' );" id="intellij-tab" data-toggle="tab" href="#">
+                <img alt="IntelliJ logo" src="{{ '/assets/images/logo/intellij-logo.png' | relative_url }}" /> IntelliJ
+            </a>
+        </li>
+    </ul>
+
+    <div class="tab-content border rounded-bottom border-top-0">
```

Comment:
For whatever reason, the vertical content border is missing for the first few pixels if the first content is a headline. This behavior can be observed on the [pr/doc/use_tabs](https://github.com/saros-project/saros/tree/pr/doc/use_tabs) branch on the "Installation" page on the eclipse tab (or see screenshot below).

![saros_page_missing_border](https://user-images.githubusercontent.com/12495415/64456442-7ed30d80-d0f0-11e9-9f1c-88fa8cd2ce21.png)